File: src/main/java/com/gtladd/gtladditions/client/render/machine/antichrist/AntichristIrisPipelineBridge.java
package com.gtladd.gtladditions.client.render.machine.antichrist;

import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.api.distmarker.OnlyIn;

@OnlyIn(Dist.CLIENT)
public interface AntichristIrisPipelineBridge {

    void beginAntichristFallbackTarget();

    void endAntichristFallbackTarget();
}

File: src/main/java/com/gtladd/gtladditions/client/render/machine/antichrist/AntichristIrisPipelineBridge.kt
package com.gtladd.gtladditions.client.render.machine.antichrist

import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.api.distmarker.OnlyIn

@OnlyIn(Dist.CLIENT)
interface AntichristIrisPipelineBridge {
    fun beginAntichristFallbackTarget()

    fun endAntichristFallbackTarget()
}